lib/rutabaga/feature.rb

```diff
@@ -3,67 +3,22 @@
 
 module Rutabaga
   module Feature
-    def feature(feature_file = find_feature)
-
+    def feature(feature_file = nil)
+      RSpec.deprecate(
+        "Calling `feature` from an `it` block",
+        :message => "Calling `feature` from an `it` block " \
+                    "is deprecated.\nIt should now be called directly in the " \
+                    "`describe` block."
+      )
+
+      feature_file = Util.find_feature(feature_file || RSpec.current_example.description)
```

Comment:
In the old syntax this will be the `it`. We could look at creating a Feature example group or something like that so you would do:

```
describe Feature do
```

Or

```
describe "run a feature", tag: :run_feature do
end

describe "a_differently_named.feature", tag: :rutabaga do
end
```

Something like that?